examples/render-helm-chart-crds/.expected/config.yaml

```diff
@@ -0,0 +1,19 @@
+# This test should be skipped because a bug in the kpt orchestrator
+# results in non-idempotent behavior.
+
+# Upon the first run, the contents of customresourcedefinition_workspaces.app.terraform.io.yaml
+# contains the following:
+
+#  creationTimestamp: null
+#  name: workspaces.app.terraform.io
+
+# On the second run, however, these fields are reversed:
+
+#  name: workspaces.app.terraform.io
+#  creationTimestamp: null
+
+# kpt should preserve the ordering of fields. Ordering is preserved if creationTimestamp
+# is anything except null (correct behavior), but is not preserved when creationTimestamp
+# is null.
```

Comment:
@natasha41575 Can you pl. file a bug for it if you haven't already. Thank you so much!

Reply:
filed https://github.com/GoogleContainerTools/kpt/issues/2462

Reply:
@natasha41575 I added the test and it works fine. https://github.com/kubernetes-sigs/kustomize/pull/4134/files

Which version of kpt are you using ? `kpt version` on your local should have version beta2 or above.

Where can I find render-helm-chart function ? I don't see it here https://pantheon.corp.google.com/gcr/images/kpt-fn/GLOBAL

Reply:
>Where can I find render-helm-chart function ?

It is being renamed in this PR. You can find it currently as `gcr.io/kpt-fn/inflate-helm-chart:unstable`

Reply:
> Which version of kpt are you using ? kpt version on your local should have version beta2 or above.

Beta3. I'll do some more debugging to see if the problem is something on my end. Thanks for adding the test!